tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -485,6 +485,20 @@ module.exports = {
       }
 
       return this.useXpath().expect.element(collaboratorSelector).to.not.be.present
+    },
+    /**
+     * @param collaborator
+     * @param value
+     *
+     * @return {Promise<*>}
+     */
+    changeCollaboratorExpiryDate: async function (collaborator, value) {
+      await collaboratorDialog.clickEditShare(collaborator)
+      await this.api.page
+        .FilesPageElement
+        .publicLinksDialog()
+        .setPublicLinkExpiryDate(value)
```

Comment:
now we are using that pageObject from the sharingDialog, so its not really part of the publicLinksDialog, we need to move it

Reply:
yes, @individual-it, I've done this in PR #3165!

Reply:
I've reviewed that PR

Reply:
@individual-it you can review again!